flink-connectors/flink-connector-jdbc/src/test/resources/mysql-scripts/catalog-init-for-test.sql

```diff
@@ -59,7 +58,7 @@ CREATE TABLE `t_all_types` (
   `col_int_unsigned` int(10) unsigned DEFAULT NULL,
   `col_integer` int(11) DEFAULT NULL,
   `col_integer_unsigned` int(10) unsigned DEFAULT NULL,
-  `col_json` json DEFAULT NULL,
+  `col_json` longtext DEFAULT NULL,
```

Comment:
The column name `col_json` and type `longtext` is not matched, we can skip the json type test by removing this field. Although this is a minor test fallback, I think it's okay as we did not cover all types like GEO type too.